WWDC Wishlist/WWDC_WishlistApp.swift

```diff
@@ -24,8 +24,8 @@ struct WWDC_WishlistApp: App {
     }()
 
     var body: some Scene {
-        WindowGroup(for: UUID.self) { id in
-            AppCoordinatorView(windowId: id)
+        WindowGroup(for: UUID.self) { _ in
```

Comment:
I initially did this because the coordinator state was being shared across windows, is that all working as expected?

Reply:
@calebwilson706 The Window ID being passed seems to cause a crash with multiple windows.

Reply:
It is not needed in this case due to the way we're using the coordinators. This seems to be a bug with Swift Data right now which can be resolved by using environment variables (but I did not see the need to pass the window ID down when we don't actually use it)

Reply:
As long as the multi tasking on iPad is working as expected